examples/csr_coo/csr_coo.cc

```diff
@@ -45,12 +45,12 @@ int main(int argc, char *argv[]) {
         file_name);
     format::COO<unsigned int, unsigned int, unsigned int> *coo =
         reader.ReadCOO();
-    auto format = coo->get_format_id();
+    auto format = coo->get_id();
     auto dimensions = coo->get_dimensions();
     auto coo_col = coo->get_col();
     auto coo_row = coo->get_row();
     auto coo_vals = coo->get_vals();
-    cout << "Format: " << coo->get_format_id().name() << endl;
+    cout << "Format: " << coo->get_id().name() << endl;
```

Comment:
We should probably change all `cout` like this to use `get_name()` over `get_id().name()` so that people aren't confused over the output.

Reply:
Not really related to this PR though so we should just remember to do this before the release.

Reply:
Good point.

Reply:
Added.